lib/src/screens/supa_phone_auth.dart

```diff
@@ -69,27 +75,44 @@ class _SupaPhoneAuthState extends State<SupaPhoneAuth> {
           ),
           spacer(16),
           ElevatedButton(
-            child: const Text(
-              'Sign Up',
-              style: TextStyle(fontWeight: FontWeight.bold),
+            child: Text(
+              signingIn ? 'Sign In' : 'Sign Up',
+              style: const TextStyle(fontWeight: FontWeight.bold),
             ),
             onPressed: () async {
               if (!_formKey.currentState!.validate()) {
                 return;
               }
-
-              try {
-                await supaAuth.signInUserWithPhone(_phone.text, _password.text);
-                if (!mounted) return;
-                successAlert;
-                if (mounted) {
-                  Navigator.popAndPushNamed(context, widget.redirectUrl,
-                      arguments: {"phone": _phone.text});
+              if (signingIn) {
+                try {
+                  await supaAuth.signInUserWithPhone(
+                      _phone.text, _password.text);
+                  if (!mounted) return;
+                  successAlert;
```

Comment:
Does writing `successAlert` actually display the alert dialog? I don't think so, but correct me if I'm wrong!

```suggestion
                  await successAlert(context);
```

Reply:
Yes it displays it

Reply:
@FatumaA 
Would you be able to double check? You would need to call the method here. Writing `successAlert` is not calling the method and therefore no dialog will be displayed here.

Reply:
ok, fixed